index.bs

```diff
@@ -188,13 +188,48 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         capabilities by a {{MediaEncodingConfiguration}} dictionary.
       </p>
       <p>
-        For a {{MediaConfiguration}} to be a <dfn>valid MediaConfiguration
-        </dfn>, <code>audio</code> or <code>video</code> MUST be <a>present</a>.
+        For a {{MediaConfiguration}} to be a <dfn>valid 
+        MediaConfiguration</dfn>, all of the following conditions MUST be true:
+        <ol>
+          <li>
+            <code>audio</code> and/or <code>video</code> MUST be <a>present</a>.
+          </li>
+          <li>
+            <code>audio</code> MUST be a <a>valid audio configuration</a> if 
+            <a>present</a>.
+          </li>
+          <li>
+            <code>video</code> MUST be a <a>valid video configuration</a> if 
+            <a>present</a>.
+          </li>
+        </ol>
+      </p>
+      <p>
+        For a {{MediaDecodingConfiguration}} to be a <dfn>valid 
+        MediaDecodingConfiguration</dfn>, all of the following conditions MUST
+        be true:
+        <ol>
+          <li>
+            It MUST be a <a>valid MediaConfiguration</a>.
+          </li>
+          <li>
+            If <code>keySystemConfiguration</code> is <a>present</a>:
```

Comment:
If none of `audioRobustness` nor `videoRobustness` is present, is it still valid?

Reply:
Yes. Both of those are optional.